Tests/x11.m

```diff
@@ -21,10 +21,10 @@ int main(int argc, char **argv)
 
   d = XOpenDisplay(NULL);
   if (!d)
-    {
-      fprintf(stderr,"Cannot open display: %s\n", XDisplayName(NULL));
-      exit(EXIT_FAILURE);
-    }
```

Comment:
The original indentation here is an example of the correct indentation for GNUstep.